crates/bevy_ui/src/render/pipeline.rs

```diff
@@ -65,6 +65,12 @@ impl SpecializedRenderPipeline for UiPipeline {
                 VertexFormat::Float32x4,
                 // mode
                 VertexFormat::Uint32,
+                // radius
```

Comment:
Add `border...` to each of these comments to make it more clear what they refer to.